lib/kernels/clink_kernels_jna.cc

```diff
@@ -12,11 +12,11 @@
  * limitations under the License.
  */
 
-#include <string>
-
 #include "clink/kernels/clink_kernels.h"
 #include "clink/utils/clink_utils.h"
 
+#include <string>
```

Comment:
Could we follow Google's C++ code style [1] when adding `#include`? In this particular case, `<string>` is C++ standard library headers which could be put before `"clink/kernels/clink_kernels.h"`.

[1] https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
OK. I'll make the change.